test/recipe-test.js

```diff
@@ -21,15 +87,15 @@ describe('Recipe', function() {
   });
 
   it('should initialize with an id', function() {
-    expect(recipe.id).to.eq(595736);
+    expect(recipe.id).to.equal(595736);
   });
 
   it('should initialize with an name', function() {
-    expect(recipe.name).to.eq('Loaded Chocolate Chip Pudding Cookie Cups');
+    expect(recipe.name).to.equal('Loaded Chocolate Chip Pudding Cookie Cups');
   });
 
   it('should initialize with an image', function() {
-    expect(recipe.image).to.eq('https://spoonacular.com/recipeImages/595736-556x370.jpg');
+    expect(recipe.image).to.equal('https://spoonacular.com/recipeImages/595736-556x370.jpg');
   });
 
   it('should initialize with an array of ingredients', function() {
```

Comment:
Looks like we might have overlooked a test. Let's get this test written with the next branch. Should be a quick fix!

Reply:
Great catch Kristi!